docs/release/releases/release-1.11.md

```diff
@@ -0,0 +1,41 @@
+# Cluster API v1.11
+
+## Timeline
+
+The following table shows the preliminary dates for the `v1.11` release cycle.
+
+| **What**                                              | **Who**       | **When**                    | **Week** |
+|-------------------------------------------------------|---------------|-----------------------------|----------|
+| Start of Release Cycle                                | Release Lead  | Monday 28th April 2025      | week 1   |
+| Schedule finalized                                    | Release Lead  | Monday 28th April 2025      | week 1   |
+| Team finalized                                        | Release Lead  | Monday 28th April 2025      | week 1   |
+| *v1.10.x released (tentative)*                        | Release Lead  | Tuesday 29th April 2025     | week 1   |
+| *v1.9.x & v1.10.x released*                           | Release Lead  | Tuesday 20th May 2025       | week 4   |
+| v1.11.0-alpha.0 released                              | Release Lead  | Tuesday 28th May 2025       | week 5   |
+| *v1.9.x & v1.10.x released*                           | Release Lead  | Tuesday 17th June 2025      | week 8   |
+| v1.11.0-alpha.1 released                              | Release Lead  | Tuesday 17th June 2025      | week 8   |
```

Comment:
```suggestion
| v1.11.0-alpha.1 released (to be confirmed)                             | Release Lead  | Tuesday 17th June 2025      | week 8   |
```
We are going to do alpha.1 and alpha.2 only if necessary (and if we have some feedback that folks actually consumes alpha releases)